app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -120,13 +120,17 @@ def textual_group_endpoints
     )
   end
 
-  def textual_group_miq_custom_attributes
-    TextualGroup.new(_("Custom Attributes"), textual_miq_custom_attributes)
+  def textual_group_miq_displayable_custom_attributes
+    TextualGroup.new(_("Custom Attributes"), textual_miq_displayable_custom_attributes)
   end
 
-  def textual_miq_custom_attributes
+  def textual_miq_displayable_custom_attributes
     attrs = @record.custom_attributes
```

Comment:
maybe instead of selecting all and then selecting only the one you want you could get only those from the db?